rosdep/python.yaml

```diff
@@ -4438,7 +4438,9 @@ python-schedule:
   ubuntu: [python-schedule]
 python-scipy:
   arch: [python2-scipy]
-  debian: [python-scipy]
+  debian:
+    '*': [python-scipy]
+    buster: [python3-scipy]
```

Comment:
@rgov that is intended. `python-scipy` is a Python 2 dependency. `python3-scipy` is the Python 3 dependency you're looking for. And it is included already.